smartrent/solicit
# CMW-2200: Response Package

## Description

![image](https://user-images.githubusercontent.com/8291834/99718872-cb420900-2a68-11eb-9714-3866d2ff1044.png)

## Steps to Test
1. Run `mix test`, and ensure tests pass
2. Review code and see if any functions that you would like in this version are missing

## Screenshots
![image](https://user-images.githubusercontent.com/8291834/99719281-591df400-2a69-11eb-8b98-fc318df22cb7.png)
